plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/META-INF/MANIFEST.MF

```diff
@@ -38,6 +38,7 @@ Import-Package: com.google.api.client.auth.oauth2;version="[1.23.0,1.24.0)",
  com.google.common.base;version="[21.0.0,22.0.0)",
  com.google.common.collect;version="[21.0.0,22.0.0)",
  com.google.common.escape;version="[21.0.0,22.0.0)",
+ com.google.common.io;version="[21.0.0,22.0.0)",
  com.google.common.net;version="[21.0.0,22.0.0)",
  javax.inject;version="1.0.0",
  org.eclipse.core.databinding,
```

Comment:
It may be possible to remove some of the imports at the end of this file that we no longer use.

 org.yaml.snakeyaml.error,
 org.yaml.snakeyaml.parser,
 org.yaml.snakeyaml.scanner

Reply:
Let's remove these unused imports from `MANIFEST.MF` too.

Reply:
.io is needed as we pull in `com.google.common.io.MoreFiles`.

Reply:
Sorry, I'm talking about

org.yaml.snakeyaml.error,
org.yaml.snakeyaml.parser,
org.yaml.snakeyaml.scanner

(GitHub doesn't allow putting comments in locations far away from changes.)